packages/itwinui-react/src/core/Checkbox/Checkbox.tsx

```diff
@@ -116,9 +127,10 @@ export const Checkbox = React.forwardRef((props, ref) => {
           [`iui-${status}`]: !!status,
           'iui-loading': isLoading,
         },
-        className,
+        wrapperClassName,
       )}
-      style={style}
+      style={wrapperStyle}
+      {...restWrapperProps}
     >
       {checkbox}
       {label && (
```

Comment:
@gretanausedaite Should we do the same thing here for label as we did for the wrapper, or is passing a custom `React.ReactNode` for `label` enough customization?

Reply:
I think it's enough. label className simply sets line-height, everything else is up to the user.
@mayank99 agree?

Reply:
no. every single element needs to be customizable.